x/twap/api.go

```diff
@@ -41,12 +41,49 @@ func (k Keeper) GetArithmeticTwap(
 	quoteAssetDenom string,
 	startTime time.Time,
 	endTime time.Time,
+) (sdk.Dec, error) {
+	arithmeticStrategy := &arithmetic{k}
+	return k.getTwap(ctx, poolId, baseAssetDenom, quoteAssetDenom, startTime, endTime, arithmeticStrategy)
+}
+
+// GetArithmeticTwapToNow returns GetArithmeticTwap on the input, with endTime being fixed to ctx.BlockTime()
+// This function does not mutate records.
```

Comment:
Generally, we should treat a function as a black box. Using a spec, we aim to convey what we get by calling the function given certain inputs.

Since twap record is an internal implementation detail that clients should not be concerned about from the black box perspective, I suggest rephrasing it

```suggestion
// GetArithmeticTwapToNow returns arithmetic twap from start time until the current block time for quote and base 
// assets in a given pool.
```